### DISCUSSION TITLE
Allow manually setting the order in the collection chooser dropdown in the builder interface

### DISCUSSION THREAD

**Discussion by schtiewen** [directus/directus#24685] [Feature Requests]
### Describe the Bug

Hi,

i've a builder with round about 30 collection. e.g. A, D, H, B, F, Z, M ... (not ordered list)
on the content page, here is the dropdown field with the same order.

is it possible to sort the output? (human readable) maybe by the collection name as a default sort order?
i would say, it make no sense, that the output dropdown has the same order like the collection field inside the setup

Thanks
:)

### To Reproduce

create a new builder field with many collections as Related Collections

### Directus Version

v11.3.2

### Hosting Strategy

Self-Hosted (Docker Image)

**Comment by rijkvanzanten** [comment]
The dropdown currently should follow the list as it's configured in the DataModel settings. Is that what you're suggesting? Or are you saying it'd be neat if there was an interface option to sort them manually?

**Comment by schtiewen** [comment]
But inside the Datamodel, there is no option to do a manually sort. If i have e.g. 10 items inside and add a new "a" collection Item, then the "a" is in on the last position. Or is this wrong and i dont know, how could i do an order? :)

I would say, the second option, (manuel sorting, or alphabetic) inside the Interface would be an option.

**Comment by rijkvanzanten** [comment]
Ahh I see what you're saying. It's now ordered by the order in which the collections where selected in the relationship settings, which isn't the most user friendly
